prep-css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    * {
+      box-sizing: border-box;
+    }
+    #card-deck {
+      display: flex;
+      flex-wrap: wrap;
+      flex-basis: calc(100% / 6);
+      padding: 24px;
+    }
+    .card-wrapper {
+      padding: 12px;
+      width: calc(100% / 6)
```

Comment:
We can remove `width` here once we move our `flex-basis`, since the default `flex-direction` of a "flex container" is `row`. In this case, the primary axis of the "flex container" will be horizontal and `flex-basis` will work like `width`.